ts/types/MIME.ts

```diff
@@ -17,12 +17,13 @@ export const VIDEO_MP4 = 'video/mp4' as MIMEType;
 export const VIDEO_QUICKTIME = 'video/quicktime' as MIMEType;
 export const LONG_MESSAGE = 'text/x-signal-plain' as MIMEType;
 
-export const isJPEG = (value: MIMEType): boolean => value === 'image/jpeg';
-export const isImage = (value: MIMEType): boolean =>
-  value && value.startsWith('image/');
-export const isVideo = (value: MIMEType): boolean =>
-  value && value.startsWith('video/');
+export const isJPEG = (value: string): value is MIMEType =>
+  value === 'image/jpeg';
+export const isImage = (value: string): value is MIMEType =>
+  !!value && value.startsWith('image/');
```

Comment:
Nitpick: we typically prefer `Boolean(foo)` over `!!foo`. Would you mind updating here and throughout this file?

For example:

```suggestion
  Boolean(value) && value.startsWith('image/');
```

Alternatively:

```suggestion
  Boolean(value && value.startsWith('image/'));
```